### ISSUE TITLE
Please update for firefox 95 (no icon in address bar).

### ISSUE THREAD

**Issue by mbousq** [Aaron-P/ToggleResistFingerprinting#7]
Hi. Thanks for the extension, I hope you would have some time to update this ... Regards.

**Comment by Aaron-P** [comment]
By default the icon goes into the tab bar, are you using an extension that modifies or replaces the tab bar, like 'vertical tabs'?

**Comment by mbousq** [comment]
Absolutely ! I get it now :). I am using sideberry extension for that + custom css to remove the tab bar.

![Screenshot from 2021-12-15 18-24-48](https://user-images.githubusercontent.com/49107491/146235696-ba05c445-6108-41c6-8b29-05d7c1bcd9b6.png)

**Comment by Aaron-P** [comment]
You will need to temporarily enable the default tab bar in order to see and move the icon.

**Comment by Aaron-P** [comment]
Were you able to revert the tab bar and find the icon? It's probably also possible to move it by editing your prefs.js but that's more involved.

**Comment by mbousq** [comment]
Yes. So its the css modification in the userChrome.css file to removes the entire tab bar the issue (as I am using vertical tabs). 
`#titlebar {
display: none !important;
}`

Wouldn't it make more sense to have the  ToggleResistFingerprinting icon on the address bar ? Next to ublock origin and so on.
